core/integration/module_shell_test.go

```diff
@@ -29,7 +29,7 @@ func daggerShell(script string) dagger.WithContainerFunc {
 
 func daggerShellNoMod(script string) dagger.WithContainerFunc {
 	return func(c *dagger.Container) *dagger.Container {
-		return c.WithExec([]string{"dagger", "--no-mod"}, dagger.ContainerWithExecOpts{
+		return c.WithExec([]string{"dagger", "-n"}, dagger.ContainerWithExecOpts{
```

Comment:
:tada: eyy this is great, I type this *so* much.

I like the ergonomics - `-n` and `-m` :smile: